core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -833,6 +833,14 @@ private[spark] object SparkSubmitUtils {
   // Exposed for testing
   var printStream = SparkSubmit.printStream
 
+  // Exposed for testing.
+  // We need to specify each component explicitly, otherwise we miss spark-streaming-kafka-0-8 and
+  // other spark-streaming utility components. Underscore is there to differentiate between
+  // spark-streaming_2.1x and spark-streaming-kafka-0-8-assembly_2.1x
+  val components = Seq("catalyst_", "core_", "graphx_", "launcher_", "mllib_", "mllib-local_",
```

Comment:
since it's in global scope now can we rename it to something more meaningful?
`IVY_DEFAULT_EXCLUDES`

Also add to the docs, since it has lost its context from the method and the docs now